model/registry/gen-ai.yaml

```
@@ -19,62 +18,55 @@ groups:
           The actual GenAI product may differ from the one identified by the client.
           For example, when using OpenAI client libraries to communicate with Mistral, the `gen_ai.system`
           is set to `openai` based on the instrumentation's best knowledge.
+
+          For custom model, a custom friendly name SHOULD be used.
+          If none of these options apply, the `gen_ai.system` SHOULD be set to `_OTHER`.
         examples: 'openai'
-        tag: llm-generic-request
       - id: request.model
         stability: experimental
         type: string
-        brief: The name of the LLM a request is being made to.
+        brief: The name of the Generative AI model a request is being made to.
```

nit: for consistency
```suggestion
        brief: The name of the GenAI model a request is being made to.
```